atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/DbReadTable.java

```diff
@@ -62,16 +64,30 @@ public DbReadTable(ConnectionSupplier conns, DbQueryFactory queryFactory) {
 
     public ClosableIterator<AgnosticLightResultRow> getAllRows(
             Iterable<byte[]> rows, ColumnSelection columns, long ts, boolean includeValues) {
+        return getAllRows(rows, columns, ts, includeValues, null);
```

Comment:
Passing null here is really ugly - as DavidS said, I'd prefer some kind of enum here.